ppr-ui/src/components/common/FormCard.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-card id="form-card" flat class="pa-8">
+    <v-row no-gutters>
+      <v-col cols="2">
+        <label class="generic-label" :class="{ 'error-text': showErrors }">{{ label }}</label>
+      </v-col>
+      <v-col class="ml-8">
+        <!-- Form information slot -->
+        <slot name="infoSlot" />
+        <!-- Form slot -->
+        <slot name="formSlot" />
```

Comment:
@orelbn As we discussed earlier regarding slots, its a super basic implementation. For an example of having a `default` or fallback template piece, see below for Business Name in the party form.

For context here, we have dozens of v-cards like this where we set a label with an initial column, with the errors etc etc. 
By using this super basic component and slotting in our forms, we don't have to do that anymore, just build out forms.

Reply:
Very cool!